src/widget/wstarrating.h

```diff
@@ -32,7 +32,9 @@ class WStarRating : public QWidget, public WBaseWidget {
     void mouseMoveEvent(QMouseEvent *event);
     void mouseReleaseEvent(QMouseEvent *event);
     void leaveEvent(QEvent *);
-    
+    virtual bool event(QEvent* pEvent);
+    void fillDebugTooltip(QStringList* debug);
```

Comment:
Erratum : something with my build cache didn't work so the tooltips are correctly displayed even with virtual.

And in wslider composed it's a public slot :/ 
https://github.com/mixxxdj/mixxx/blob/master/src/widget/wslidercomposed.h#L52-L55